.gometalinter.json

```diff
@@ -5,5 +5,6 @@
     "Deadline": "120s",
     "Aggregate": true,
     "Sort": ["severity", "path", "line", "message"],
-    "Checkstyle": false
+    "Checkstyle": false,
+    "Cyclo": 12
```

Comment:
Better skip function cyclo check. I will try to find the comment that needs to be added to disable the check

Reply:
https://github.com/alecthomas/gometalinter#comment-directives